examples/datalog/stratifier.flix

```diff
@@ -0,0 +1,88 @@
+namespace Stratifier {
+    type alias Program = List[Constraint]
```

Comment:
Type aliases are just that. Why not have real enums? To cumbersome? but the example are small..

Reply:
I used opaque types but that caused some of the errors i repported with type classes